core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/textannotation/View.java

```diff
@@ -559,4 +558,9 @@ public int count() {
         return this.constituents.size();
     }
 
+    @Override
+    public Object clone() throws CloneNotSupportedException {
+        View view = (View) super.clone();
```

Comment:
Can we discuss this once tomorrow? Some concerns about cloning views and constituents.